R/make_shortcuts.R

```diff
@@ -62,13 +61,14 @@ make_shortcuts <- function(file = "R/shortcuts.R") {
 
          roxygen_blocker#' {fn_title}
          roxygen_blocker#'
-         roxygen_blocker#' You must have the `bcmapsdata` package installed to use this function.
          roxygen_blocker#'
          roxygen_blocker#' @param class what class you want the object in? `\"sf\"` (default) or `\"sp\"`.
+         roxygen_blocker#' @param ask Should the function ask the user before downloading the data to a cache?
+         roxygen_blocker#' Defaults to the value of interactive()
          roxygen_blocker#'
          roxygen_blocker#' @return The spatial layer of `{fn_name}` in the desired class
          roxygen_blocker#'
-         roxygen_blocker#' @details Type `?bcmapsdata::{fn_name}` for details.
+         roxygen_blocker#' @source `bcdata::{make_bcdata_fn(fn_title)}`
```

Comment:
This is very clever :)

Reply:
> Should we now use the high res bc boundary from the catalogue with get_catalogue_data() and cache it, instead of the StatsCan one?

That would be this [one](https://catalogue.data.gov.bc.ca/dataset/province-of-british-columbia-boundary-terrestrial)?

> There is some redundancy/inconsistency in use_data() between the individual process_[xyz].R and create_internal_data.R

I was wondering if we could somehow remove most of this anyway. For example if we had this code _inside_ the `bc_bound` function and just cache it once somehow use it?  It would help keep the package itself very small. 


> Should we have some sort of makefile-like process to build the internal data and create the shortcuts etc?

That would be great. I think it depends on how much internal data we end up having. Are you thinking an actual make file?